apps/gitlab/src/connectors/users.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * DISCLAIMER:
+ * This is an example connector, the function has a poor implementation. When requesting against API endpoint we might prefer
+ * to valid the response data received using zod than unsafely assign types to it.
+ * This might not fit your usecase if you are using a SDK to connect to the Saas.
+ * These file illustrate potential scenarios and methodologies relevant for SaaS integration.
+ */
+
+import { env } from '@/env';
+import { GitlabError } from './commons/error';
+import { getNextPageFromLink } from './commons/pagination';
+
+export type GitlabUser = {
+  id: string;
+  username: string;
+  name: string;
+  email: string | undefined;
+};
+
+type GitlabResponse = GitlabUser[];
+
+export type GitlabPaginatedResponse<T> = {
+  'next'?: string;
+  data: T[];
+};
+
+export type GetUsersParams = {
+  token: string;
+  page: string | null;
+};
+export const getUsers = async ({ token, page }: GetUsersParams) => {
+
+  const url = new URL(`${env.GITLAB_API_BASE_URL}api/v4/users`);
+  url.searchParams.append('pagination', 'keyset');
+  url.searchParams.append('per_page', '2');

```

Comment:
```suggestion
  url.searchParams.append('per_page', env.USERS_SYNC_BATCH_SIZE);
```

in env.ts:
```
USERS_SYNC_BATCH_SIZE: z.coerce.number().int().positive().default(2)
```
Could you update the default value with the max value that the gitlab api allows ?

Reply:
Sure, I will do.